src/main/java/dog/pawbook/model/managedentity/dog/Sex.java

```diff
@@ -1,21 +1,32 @@
 package dog.pawbook.model.managedentity.dog;
 
+import static dog.pawbook.commons.util.AppUtil.checkArgument;
 import static java.util.Objects.requireNonNull;
 
 public class Sex {
 
+    public static final String MESSAGE_CONSTRAINTS = "Sex should only be M/Male or F/Female";
+    public static final String VALIDATION_REGEX = "^male$|^female$|^Female$|^Male$|^M$|^F$";
```

Comment:
perhaps there's a more succinct way to express this, but i think it's fine for now